### DISCUSSION TITLE
ebusctl vs mqtt

### DISCUSSION THREAD

**Discussion by henkeberggren** [john30/ebusd#746] [Q&A] [answered]
Hello, If i run `ebusctl read -f CurrentYieldPower` i get a response with a value. But when ebusd respond to my node-red flow the value is empty. I get other values from node-red. What could be wrong?

Screen shoots

Terminal:
![image](https://user-images.githubusercontent.com/2746017/206569895-0f817406-1ca4-45ad-93b6-82b5451bf589.png)

Node-Red:
![image](https://user-images.githubusercontent.com/2746017/206570015-ef3abda6-f23b-4d01-8db0-6843e38ae9ee.png)

```
root@ba739afb0b96:/# ebusctl info                     
version: ebusd 22.4.v22.4
update check: OK
device: 192.168.1.143:3333
signal: acquired
symbol rate: 53
max symbol rate: 589
min arbitration micros: 0
max arbitration micros: 3822
min symbol latency: 1
max symbol latency: 89
reconnects: 0
masters: 5
messages: 563
conditional: 0
poll: 1
update: 10
address 00: master #1
address 03: master #11
address 05: slave #1, scanned "MF=Vaillant;ID=VR921;SW=2706;HW=5703"
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0308;HW=0403", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=BASV2;SW=0507;HW=1704", loaded "vaillant/15.basv.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd, scanning
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZ00;SW=0308;HW=0403"
address ec: slave, scanned "MF=Vaillant;ID=SOL00;SW=0507;HW=1704", loaded "vaillant/ec.sol.sc.csv"
```

**Comment by m0deller** [comment]
I'm using MQTT with node-red and confirm that I receive the msg as: ebusd/hmu/CurrentYieldPower
I listen on ebusd/# and use a switch statement to filter. Hope this helps.

**Comment by henkeberggren** [comment, marked as answer]
I'm also listening to ebusd/hmu/#  and other values is displayed. But no values for CurrentConsumedPower and CurrentYieldPower.

**CurrentConsumedPower**
![image](https://user-images.githubusercontent.com/2746017/206998000-8014f357-9ff3-4776-8c8b-4ede5c5d71af.png)

**CurrentYieldPower**
![image](https://user-images.githubusercontent.com/2746017/206998495-d4b91ac2-0608-404e-9da5-94c4ae7782d3.png)

**Reply by m0deller** [reply]
Are you actively polling for it? I send a get every 60seconds to retrieve the value. It's not broadcasted.

**Reply by henkeberggren** [reply]
Yes I'm actively poll for it via node-red.

**Reply by henkeberggren** [reply]
Ok, the pump reports 0 when idle. I thought it always report some small usage.... Topic closed.
